core/src/main/java/de/danoeh/antennapod/core/storage/DownloadRequester.java

```diff
@@ -203,25 +203,26 @@ public synchronized void downloadFeed(Context context, Feed feed, boolean loadAl
             args.putBoolean(REQUEST_ARG_LOAD_ALL_PAGES, loadAllPages);
 
             DownloadRequest request = createRequest(feed, null, new File(getFeedfilePath(), getFeedfileName(feed)),
-                    true, username, password, lastModified, true, args);
+                    true, username, password, lastModified, true, args, initiatedByUser
+            );
             if (request != null) {
                 download(context, request);
             }
         }
     }
 
     public synchronized void downloadFeed(Context context, Feed feed) throws DownloadRequestException {
-        downloadFeed(context, feed, false, false);
+        downloadFeed(context, feed, false, false, true);
     }
 
-    public synchronized void downloadMedia(@NonNull Context context, FeedItem... feedItems)
+    public synchronized void downloadMedia(@NonNull Context context, boolean initiatedByUser, FeedItem... feedItems)
             throws DownloadRequestException {
-        downloadMedia(true, context, feedItems);
+        downloadMedia(true, context, initiatedByUser, feedItems);
 
     }
 
     @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
-    public synchronized void downloadMedia(boolean performAutoCleanup, @NonNull Context context,
+    public synchronized void downloadMedia(boolean performAutoCleanup, @NonNull Context context, boolean initiatedByUser,
```

Comment:
Please break to a new line. The maximum line length should be 120 characters. This is currently not enforced by the CI because other parts of the app do not comply but I want new code to be consistent.

Reply:
Changed.